### PR TITLE
[WIP]Add some features after fork

### PR DESCRIPTION
**Pull request by Wescoeur**

### CODE REVIEW

File: usr/bin/qemuback.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 from xapi.storage.libs.util import var_run_prefix
 import sys
 
+
```
Comment (benjamreis) [inline usr/bin/qemuback.py:9]: Why 2 empty lines?

File: usr/lib/python2.7/site-packages/xapi/storage/libs/qemudisk.py [hunk 2]
```diff
@@ -1,16 +1,20 @@
...
+import re
+import signal
 import subprocess
-import xen.lowlevel.xs
+import time
+
+import cPickle
 
 # TODO: Get rid of 'image' module
 # import image
```
Comment (benjamreis) [inline usr/lib/python2.7/site-packages/xapi/storage/libs/qemudisk.py:12]: Import is commented, is the TODO still relevant?

File: usr/lib/python2.7/site-packages/xapi/storage/libs/libcow/datapath.py [hunk 3]
```diff
@@ -303,14 +273,12 @@ class QdiskDatapath(COWDatapath):
     @staticmethod
     def parse_uri(uri):
         # uri will be like:
-        # "vhd+tapdisk://<sr-type>/<sr-mount-or-volume-group>|<volume-name>"
+        # "qdisk://<sr-type>/<sr-mount-or-volume-group>|<volume-name>"
```
Comment (benjamreis) [inline usr/lib/python2.7/site-packages/xapi/storage/libs/libcow/datapath.py:276]: You always replaced `vhd+tapdisk` by `tapdisk` in this PR, here `qdisk`, why?

File: usr/bin/qemuback.py (hunk 1 above)
Comment (Wescoeur) [reply usr/bin/qemuback.py:9]: Because I use a python linter. :)

File: usr/lib/python2.7/site-packages/xapi/storage/libs/qemudisk.py (hunk 2 above)
Comment (Wescoeur) [reply usr/lib/python2.7/site-packages/xapi/storage/libs/qemudisk.py:12]: It's in the original XS7.5, but yes I don't understand this comment.
We can ignore this comment for the moment but we will check all files in the future to understand all behaviors.

Comment (benjamreis) [reply usr/lib/python2.7/site-packages/xapi/storage/libs/qemudisk.py:12]: Agreed.

File: usr/lib/python2.7/site-packages/xapi/storage/libs/libcow/datapath.py (hunk 3 above)
Comment (Wescoeur) [reply usr/lib/python2.7/site-packages/xapi/storage/libs/libcow/datapath.py:276]: It's an error in the imported code.
The `libcow` contains code relative to Qcow2, QEMU... The uri scheme name must be `qdisk`, not `vhd+tapdisk`.

**Pull request merged**